### DISCUSSION TITLE
Trigger car status update explicitly

### DISCUSSION THREAD

**Discussion by prog1407** [teslamate-org/teslamate#2108] [Q&A]
Hello everyone!
First of all thanks a lot for Teslamate, the project is awesome!

I would like to check if there is a way to explicitly force teslamate to check car status after it was asleep/offline.
Right now I start the car at home, start driving and see that sometimes gps tracking begins only in 100-200 meters from home.
I assume that it is due to a fact that while car is asleep -- teslamate is polling for updates once in some period (30 seconds?). I guess in my case teslamate misses the real moment when car becomes online and I start driving. So it detects it a bit later due to a polling period time lag.

For me it is quite important to recognize that car is online and get gps data from very beginning of the ride. Because I use that gps data in my smart home automation for closing garage gate once the car is outside.

So my idea is to catch car online action with Tasker by detecting bluetooth connection between phone and Tesla. Tasker then can send some API request and notify teslamate that car status needs to be polled and updated asap. 
Is there any kind of such API within teslamate?

Thanks a lot in advance!

**Comment by LelandSindt** [comment]
@prog1407 have a look at this discussion.

adriankumpf/teslamate#1433
